src/app/stack/stack-details/stack-details.component.ts

```diff
@@ -107,6 +108,7 @@ export class StackDetailsComponent implements OnChanges, OnInit {
      */
     public handleModalClose(): void {
         this.resetFields();
+        SaveState.isModalOpened = 0;
```

Comment:
Could you move this to resetFields

Reply:
Since you have used isModalOpened, I think it makes sense to have true / false. What do you think?

Reply:
moving this to reset Fields does not work here. since this has to be switched off, it is better we switch it off on modal close.

Reply:
I can convert the variable to boolean. let me do and check